static/xml/toolbox.xml

```diff
@@ -451,7 +451,20 @@
                     </value>
                 </block>
                 <block type="loader"></block>
-                <block type="block_holder"></block>
+                <block type="block_holder"></block><block type="webhook">
+                    <value name="WEBHOOK_URL">
+                        <shadow type="text">
+                            <field name="TEXT">https://example.com</field>
+                        </shadow>
+                    </value>
+                </block>
+                <block type="webhook_payload">
```

Comment:
We could add a `webhook_payload` shadow block here, this will make it easier for the user to understand that one must be attached.